packages/terra-table/src/SelectableUtils.js

```diff
@@ -1,7 +1,20 @@
 /**
-* Returns the index of the first selected row for SingleSelectableRows.
-* To be used in the constructor, to set initial state.
-*/
+ * Returns a validated max count for selection based on the rows and max provided.
```

Comment:
nit: 'Returns a validated max count for selection. Validates the max count prop, and if undefined returns a max of the count of children.'